charts/featureform/charts/ingress/templates/http-ingress.yaml

```diff
@@ -18,8 +19,10 @@ spec:
       port:
         number: 80
   rules:
+    {{ if not .Values.global.hostname }}
```

Comment:
should this be `{{ if .Values.global.hostname }}`?

Reply:
+1 looks like a typo. 
otherwise the rule reads: if no hostname value, use the <empty> hostname value